File: Praktikum 5. Function/Soal 1/PRAK501-2310817310001-AvantioFierzaPatria.c
#include <stdio.h>

// Deklarasi fungsi
int MaxBilangan(int a, int b, int c, int d)
{
    int max = a;

    if (b > max)
    {
        max = b;
    }
    if (c > max)
    {
        max = c;
    }
    if (d > max)
    {
        max = d;
    }

    return max;
}

int main()
{
    int a, b, c, d;
    scanf("%d %d %d %d", &a, &b, &c, &d);

    // Panggil fungsi MaxBilangan dan simpan hasilnya di variabel 'hasil'
    int hasil = MaxBilangan(a, b, c, d);

    printf("%d", hasil);

    return 0;
}

File: Praktikum 5. Function/Soal 1/PRAK501-2310817310001-AvantioFierzaPatria.py
def max_bilangan(a, b, c, d):
    max_val = a

    if b > max_val:
        max_val = b
    if c > max_val:
        max_val = c
    if d > max_val:
        max_val = d

    return max_val


# Input dari pengguna
a, b, c, d = map(int, input().split())

# Panggil fungsi max_bilangan dan simpan hasilnya di variabel 'hasil'
hasil = max_bilangan(a, b, c, d)

# Cetak hasil
print(hasil)
